app/celery/scheduled_tasks.py

```diff
@@ -533,3 +535,27 @@ def weekly_dwp_report():
         ),
         due_at=convert_utc_to_bst(datetime.utcnow()) + timedelta(days=7),
     )
+
+
+@notify_celery.task(bind=True, name="change-dvla-password", max_retries=3, default_retry_delay=60)
+def change_dvla_password(self):
+    try:
+        dvla_client.change_password()
+    except LockError:
+        # some other task is currently changing the api key. let that process handle it and quietly exit
+        current_app.logger.info("change-dvla-api-key lock held by other process, doing nothing")
```

Comment:
Just a tiny thing, but it looks like this log line (and the one a few lines below) should start with `change-dvla-password` and not `change-dvla-api-key`

Reply:
good spot, squashed